developer_setup/plugins.md

```diff
@@ -36,6 +41,23 @@ git clone git@github.com:JoeSmith/manageiq-content.git plugins/manageiq-content
 git clone git@github.com:JoeSmith/manageiq-ui-classic.git plugins/manageiq-ui-classic
 ```
 
+#### Setting the correct remotes
+
+Inside your checkout you will have the default remote origin pointing to your fork. Now add an upstream remote which
+points to the upstream repository. To keep your fork up to date, you will have to merge changes from upstream
+into your fork.
+
+```bash
+cd manageiq-ui-classic
+git remote -v
+# origin  git@github.com:JoeSmith/manageiq-ui-classic.git (fetch)
+# origin  git@github.com:JoeSmith/manageiq-ui-classic.git (push)
+git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic
+git fetch upstream
+# merge upstream into your fork
+git merge upstream/master && git push origin
```

Comment:
The only thing I push to my fork on Github is feature branches.

I guess I don't see the point in keeping master in sync on my fork especially if that's the only branch being kept in sync (none of the release branches or tags are kept in sync).